context.go

```diff
@@ -389,7 +389,10 @@ func (c *context) FormFile(name string) (*multipart.FileHeader, error) {
 	if err != nil {
 		return nil, err
 	}
-	f.Close()
+	err = f.Close()
```

Comment:
Have you checked in which situation this could error out? Just to be sure that we do not introduce some "unexpected" which was ok to be ignored.